src/app/update-profile-page/update-profile/update-profile.component.html

```diff
@@ -61,7 +61,7 @@
           style="background-repeat: no-repeat; background-position: center; background-size: cover"
         >
           <div>
-            <div *ngIf="profilePicInput != null && profilePicInput.length > 0">
+            <div *ngIf="profilePicInput !== null && profilePicInput.length > 0">
```

Comment:
Another thing I noticed when running the changes locally is many things get broken by changing double equals to triple equals since it changes the behavior to be more strict. Comparison to `null` using `==` will also match with `undefined` but changing to use `===` will ONLY match to `null`. I noticed this broke a lot of things on the signup and update profile pages when the user doesn't have a profile yet. 

In order to fix this in general, we can use the `isNil` function from lodash, which under the hood just does a double equal check against null but it won't trigger the linter since it's buried in lodash.

Reply:
@jackson-dean thank you so much for catching these things for me. I have to say I expected something like this to break because such changes from `==` to `===` are always too risky.

I re-reviewed all the not-`Viewed` files in this PR and made sure to check every place that was changed from `==` to `===`. I left untouched all the places where we used to check for `.length === 0` or checking for equality with some enums or constant values. The rest of the changes were refactored.

Please let me know if it makes sense to you.